.changeset/six-turkeys-type.md

```diff
@@ -0,0 +1,6 @@
+---
+'@itwin/itwinui-react': minor
+'@itwin/itwinui-css': patch
+---
+
+`Anchor` can now be rendered as a button using `as` prop.
```

Comment:
Do you want to include an example?

Reply:
added. [`aae2d95`](https://github.com/iTwin/iTwinUI/pull/1006/commits/aae2d9595478affb465a0f268ca10424abb29e21)